OpenRA.Game/WAngle.cs

```diff
@@ -79,6 +79,27 @@ public static WAngle Lerp(WAngle a, WAngle b, int mul, int div)
 			return new WAngle(aa + (bb - aa) * mul / div);
 		}
 
+		public static WAngle ChangeByStep(WAngle a, WAngle b, WAngle step_speed)
```

Comment:
Can you please instead define this as a `TickOrientation` method in the Mods.Common `Utils` class - to match `TickFacing`.